benchmarks/encoder/config.yaml

```diff
@@ -21,3 +22,11 @@ one-metric:
 many-metrics:
   <<: *base_variant
   nmetrics: 48
+dd-origin:
+  <<: *base_variant
+  ntags: 10
+  ltags: 16
+  dd_origin: true
+no-tags-with-dd-origin:
```

Comment:
maybe we should name `one-trace-with-dd-origin` to match the baseline + dd-origin?

not a strong opinion